acceptance/bundle/test.toml

```diff
@@ -1,6 +1,3 @@
-Local = true
```

Comment:
Is local true by default?

Reply:
Yes, Local = true, Cloud = false set in https://github.com/databricks/cli/blob/main/acceptance/test.toml